### ISSUE TITLE
See chemphil.html in inputs

### ISSUE THREAD

**Issue by tarahmarie** [jdmartin/gutenberg-text-splitter#39]
Attempting to process shows no useful outputs after analyzing and seeing samples. It's useful to have an anti-example, perhaps. Or maybe there's some logic that can look for longer text, find that heading, and include it in the samples option?

**Comment by jdmartin** [comment]
Yeah, this one's got some really odd nested stuff.  (chap gets used as a class whenever a chapter is mentioned, and not when a chapter is begun.)  h3 is a promising choice, but it is also inconsistent.  I'll see what I can find.

**Comment by jdmartin** [comment]
This is the crux of my problem:

```html
<div class="chapter">
<p><span class="pagenum" id="Page_xi">[Pg xi]</span></p>
<h2 class="nobreak">INTRODUCTION.</h2>
</div>
<p class="drop-cap">
```

A chapter is announced inside a div that closes before the content begins.  You might think, in looking at this, that we can use the drop-cap... but, of course, that's only at the head of a large division.  The smaller sections have a `<span>` with a "smcap" class, but these aren't consistent either.

Of course, if the ToC were marked, I might be able to make some inferences from the anchor tags, (each subsection is linked, and the anchors are sequentially numbered) but these are also peppered through the document.  So, I'm not really sure what to do with this, if we're being perfectly honest.

At the moment, using `<h3>` does divide my text into helpful sections.  These correspond to the large divisions of the text.  There might be a way to post-process, but I don't see any likely targets that are consistent in those divisions apart from, perhaps, the `<p>` which begin with sequential ids.  Will have to look at this as a longer term issue.
